htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/element/Table.java

```diff
@@ -142,7 +142,10 @@ public WebElement getCellAt(int i, int j) {
      * Returns list of maps where keys are table headings and values are table row elements ({@code <td>}).
      */
     public List<Map<String, WebElement>> getRowsMappedToHeadings() {
-        return getRowsMappedToHeadings(getHeadingsAsString());
+        return getRows().stream()
+                .map(row -> row.stream()
+                        .collect(toMap(e -> getHeadingsAsString().get(row.indexOf(e)), identity())))
```

Comment:
Yes, my bad, made another bug by fixing the original one :( Thank you! I've made another pull request with a fix as per your suggestion.